res/values-pt-rBR/strings.xml

```diff
@@ -471,7 +473,7 @@
     <string name="ENTER">"ENTRAR"</string>
     <string name="_1v1">"1v1"</string>
     <string name="Practice">"Praticar"</string>
-    <string name="ArenaDescription">Você terá apenas uma vida. O vencedor receberá %s plasma. Negociar vitórias ou "pontos gratuitos" resultará em um banimento.</string>
+    <string name="ArenaDescription">Você terá apenas uma vida. O vencedor receberá %s plasma. Negociar vitórias ou "pontos gratuitos" resultarão em um banimento.</string>
```

Comment:
If it was "Negociar vitórias **e** pontos gratuitos", the **resultarão** would be right, but what happens is not that.